tests/security/authz_td_migration/scripts/authz_td_migration.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -u
+set -o pipefail
+
+source "${REPO_ROOT}/content/en/docs/tasks/security/authorization/authz-td-migration/snips.sh"
+source "${REPO_ROOT}/tests/util/samples.sh"
+
+max_attempts=10
```

Comment:
I'm removing the `max_attempts` argument from the `_run_and_verify` functions in #7440. You can override the default 5 attempts by setting the `VERIFY_RETRIES` environment variable.

Reply:
Sounds good.  This test is particularly slow, so it needs a higher retry count.  Hopefully we can start rolling out tests with the new istioctl "wait" mechanism to improve this situation.